docs/MigrationGuide.md

```diff
@@ -111,6 +112,7 @@ To see the biggest differences please consult the [changelog](Changelog.md).
     * `SingleGenericContextTransformationWrapper` to `SingleInputDynamicComponentWrapper`
     * `SourceFactoryGenericContextTransformationWrapper` to `SourceFactoryDynamicComponentWrapper`
     * `JoinGenericContextTransformationWrapper` to `JoinDynamicComponentWrapper`
+  * type `NodeTransformationDefinition` (inside `DynamicComponent`) renamed to `ContextTransformationDefinition`
```

Comment:
what's this?

Reply:
Missing migration guide entry, that will be helpful when upgrading Nu to 1.14